<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>checkout</class>
 <widget class="QDialog" name="checkout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextBrowser" name="receiptBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>21</y>
     <width>481</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>210</width>
     <height>56</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="cardNumber_label">
        <property name="text">
         <string>Card Number:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="cardNumber_line">
        <property name="placeholderText">
         <string extracomment="Enter Card Number."/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="zipCode_label">
        <property name="text">
         <string>Zip Code:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="zipCode_line">
        <property name="placeholderText">
         <string extracomment="Enter your ZIP."/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>-60</x>
     <y>-20</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2"/>
  </widget>
  <widget class="QPushButton" name="purchase">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Complete Purchase</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>310</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel Checkout</string>
   </property>
  </widget>
  <widget class="QLabel" name="purchase_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Purchase Details:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>210</y>
     <width>164</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>CVV:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="CVV_line">
      <property name="placeholderText">
       <string extracomment="Enter the CVV."/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
